### ISSUE TITLE
edgedb create-migration erorr message

### ISSUE THREAD

**Issue by 1st1** [edgedb/edgedb-cli#258]
```
~/d/e/bench » edgedb -I bench create-migration
edgedb error: could not read schema in ./dbschema: could not read schema file ./dbschema/schema.esdl: final statement does not end with a semicolon
```

The `edgedb error: could not read schema in ./dbschema: could not read schema file` is hard to parse/read.  It should say something along these lines:

```
~/d/e/bench » edgedb -I bench create-migration
edgedb error: could not read schema file ./dbschema/schema.esdl. 

Syntax error: final statement does not end with a semicolon:

   module default {
       type Blah { ... }
       type Blah { ... }
                       ^-- ";" is missing
   }
```

**Comment by 1st1** [comment]
Actually, this is weird. If I put the semicolon there it crashes correctly, just in the way I suggested:

```
~/d/e/bench » edgedb -I bench create-migration
error: Unexpected ':'
  ┌─ ./dbschema/schema.esdl:4:23
  │
4 │             constraint: exclusive;
  │                       ^^ error
```

So why do we have the "final statement does not end with a semicolon" error? Where does it come from?

We do not require trailing `;` after last blocks in our SDL grammar. Why does the CLI want it? This looks like a bug.

**Comment by 1st1** [comment]
cc @vpetrovykh

**Comment by vpetrovykh** [comment]
Perhaps this is legacy from the time that migrations were handled entirely by DDL. Then it was fair to say that a DDL file *must* end in `;`.

**Comment by elprans** [comment]
Absolutely coincidentally I'm looking at the semicolon situation in SDL right now.

**Comment by tailhook** [comment]
We do require them, because otherwise concatenating files can be ambiguos (note: we concatenate them in arbitrary order).

**Comment by 1st1** [comment]
> We do require them, because otherwise concatenating files can be ambiguos (note: we concatenate them in arbitrary order).

Alright. Hopefully https://github.com/edgedb/edgedb/pull/2302 will resolve this issue.

**Comment by tailhook** [comment]
Fixed by#276
